pkg/vgmanager/vgmanager_controller.go

```diff
@@ -324,6 +324,10 @@ func (r *VGReconciler) processDelete(ctx context.Context, volumeGroup *lvmv1alph
 	}
 	if lvmdConfig == nil {
 		r.Log.Info("lvmd config file does not exist")
+		if statuserr := r.updateStatus(ctx, nil); statuserr != nil {
```

Comment:
Please add a comment here to explain why this is being done.

Reply:
Updated.

Reply:
I think we can explain more

Reply:
looks good now